modules/InboundEmail/InboundEmail.php

```diff
@@ -5000,6 +5008,115 @@ public function returnImportedEmail($msgNo, $uid, $forDisplay = false, $clean_em
         return true;
     }
 
+    public function returnNonImportedEmail($msgNo, $uid)
```

Comment:
could you add some valid PHPDoc to this function too please? I'm asking it three different reason because I'm not really sure what the arguments mean, I don't know what the possible return values mean because this function too long